.github/workflows/ci.yml

```diff
@@ -168,11 +186,8 @@ jobs:
 
       - name: Publish
         env:
-          NEXUS_UPD_ID: ${{ secrets.RELEASE_NEXUS_UPD_ID }}
-          NEXUS_UPD_PASS: ${{ secrets.RELEASE_NEXUS_UPD_PASS }}
-          SIGNING_KEY_ID: ${{ secrets.RELEASE_SIGNING_KEY_ID }}
-          SIGNING_KEY: ${{ secrets.RELEASE_SIGNING_KEY }}
-          SIGNING_KEY_PASS: ${{ secrets.RELEASE_SIGNING_KEY_PASS }}
+          MAVEN_USER: ${{ secrets.RELEASE_MAVEN_USER }}
+          MAVEN_TOKEN: ${{ secrets.RELEASE_MAVEN_TOKEN }}
```

Comment:
please revert, this is specific to publishing to my maven and shouldn't be merged